src/test/java/org/apache/datasketches/theta/CornerCaseThetaSetOperationsTest.java

```diff
@@ -20,6 +20,8 @@
 package org.apache.datasketches.theta;
 
 import org.testng.annotations.Test;
+//import static org.apache.datasketches.Util.DEFAULT_UPDATE_SEED;
+//import static org.apache.datasketches.hash.MurmurHash3.hash;
 
```

Comment:
do we need to keep these commented-out lines?

Reply:
These two imports are associated with some helper methods at the bottom of the file, which are also commented out.  These helper methods would aid a reader in understanding how the various test hash values used in all the tests are actually generated.